React/tic-tac/src/app/screens/Login/components/Form.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import validate from '../validate.js';
+import style from '../style.scss';
+
+const renderField = ({ input, label, className, type, meta: { touched, error } }) => (
+  <div className={style.field}>
+    <label htmlFor="{label}">{label}</label>
+    <div>
+      <input {...input} placeholder={label} type={type} className={className} />
+      {touched && (error && <span className={style.alert}>{error}</span>)}
+    </div>
+  </div>
+);
+
+const SyncValidationForm = props => {
```

Comment:
You named this component folder as `Form` but the component definition itself is named `SyncValidationForm`. Try to keep consistent names so it'll be easier for you to understand how your code behaves. I'd recommend using `LoginForm` as the name for this component, but if you think of a more descriptive name, go for it! 🙂